typescript/eventbridge-lambda/README.md

```diff
@@ -0,0 +1,35 @@
+# EventBridge Rule scheduled Lambda
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+> **This is a stable example. It should successfully build out of the box**
+>
+> This examples is built on Construct Libraries marked "Stable" and does not have any infrastructure prerequisites to build.
+
+---
+<!--END STABILITY BANNER-->
+
+This example creates a new EventBridge rule that executes a Lambda function every day at 6pm UTC
+
+
+## Build
+
+To build this app, you need to be in this example's root folder. Then run the following:
+
+```bash
+npm install -g aws-cdk
+npm install
+npm run build
+```
+
+This will install the necessary CDK, then this example's dependencies, and then build your TypeScript files and your CloudFormation template.
+
+## Deploy
+
+Run `cdk deploy`. This will deploy / redeploy your Stack to your AWS Account.
+
```

Comment:
And how do you know it worked? How does one see that the lambda function is being called?

Reply:
Addressed in latest commit with sns topic subscription so you can get confirmation that the stack is working